uber/custom_tags.py

```diff
@@ -130,6 +130,19 @@ def urlencode(s):
 
 
 @JinjaEnv.jinja_filter
+def url_to_link(url=None, text=None, target=None):
+    # Jinja2 has a 'urlize' filter but it depends on links having `.com` or `http` in the name
+    # This works with relative links and allows you to also define link text
+    if not url:
+        return ''
+
+    if not text:
+        text = url
+
+    return safe_string('<a href="{}" {}>{}</a>'.format(url, 'target="{}"'.format(target) if target else '', text))
```

Comment:
We have to be a little careful here. We probably want to urlencode `url`, and html escape both `target` and `text`.

And we definitely need to write some tests that pass bad data (like `url` with quotes in it, and `text` with `<` and `>`, basically data with all the special characters).